server/pkg/slice/ints.go

```diff
@@ -11,14 +11,18 @@ type (
 	UInt64s []uint64
 )
 
-func HasUint64(ss []uint64, s uint64) bool {
-	for i := range ss {
-		if ss[i] == s {
-			return true
+func HasUint64(ss []uint64, s uint64) (bool, int) {
```

Comment:
does this need a bool and an int? -1 is an invalid index which could be used to replace the bool `false`